src/data.toml

```diff
@@ -1,18 +1,22 @@
 enable_experiments = "Head to Settings > Bunny > Activate Discord Experiments"
 account_switcher = "Enable Discord Experiments and look for the 'Mobile Account Switcher' experiment and set it to Treatment 1. Hold your profile in the navigation tab to manage your accounts."
 disable_nmu = "The only way to disable NMU is to use Discord version <= 224 and change the experiment's treatment. Keep in mind that support is only provided for the latest stable release and your version may stop working at some point."
-fix_voice_activity = "The cause of this is unclear. To fix it, please disable 'Advanced Voice Activity' in voice settings."
-no_mod_settings = "Try visiting [this link](<https://raw.githubusercontent.com/pyoncord/detta-builds/main/bunny.js>) and see if it loads bunch of texts. If it doesn't, try using a VPN and/or disable any sort of ad blocker and try again.\n-#These troubleshooting steps may still resolve your issue even if the link loads a bunch of text."
+fix_voice_activity = "The cause of this is unclear. To fix it, please disable `Advanced Voice Activity` in voice settings."
+no_mod_settings = "Try visiting [this link](<https://raw.githubusercontent.com/pyoncord/detta-builds/main/bunny.js>) and see if it loads bunch of texts. If it doesn't, try using a VPN and/or disable any sort of ad blocker and try again.\nThese troubleshooting steps may still resolve your issue even if the link loads a bunch of text."
 custom_fonts = "Fonts are experimental and some packs may break the application. You can get font packs listed here: https://discord.com/channels/1196075698301968455/1225584192012947506/1241453802184380418"
 ios_notifications = "For notifications to work on a Jailed device you need to sign the ipa with a certificate that has the notification entitlement enabled.\n\nPlease keep in mind that this channel is not sideloading support.\nPlease keep questions pertaining to this topic elsewhere."
-thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\nYou may use a system information app such as `CPU-Z` or `AIDA64` to check the architecture of your CPU and operating system."
+thirty_two_bit_android_os = "32-bit versions of Android OS are not supported. Please use a 64-bit operating system.\nYou may use a system information app such as `CPU-Z` or `AIDA64.` to check the architecture of your CPU and operating system."
```

Comment:
why? the sentence isn't finished at that point

Reply:
I just pushed whatever was changed by Nexpid without any reviewing. @nexpid Please explain your changes.